Goutte/Client.php

```diff
@@ -112,7 +125,7 @@ protected function doRequest($request)
                 $requestOptions['body'] = $content;
             } else {
                 if ($files = $request->getFiles()) {
-                    $requestOptions['multipart'] = [];
+                    $requestOptions['multipart'] = array();
```

Comment:
there is no need to revert bakc to the long array syntax. This package [requires PHP `>=5.5.0`](https://github.com/FriendsOfPHP/Goutte/blob/master/composer.json#L15)